Sources/SwiftIfConfig/BuildConfiguration.swift

```diff
@@ -114,15 +115,15 @@ public protocol BuildConfiguration {
   /// information, which will translate into the `version` argument.
   ///
   /// - Parameters:
-  ///   - importPath: A nonempty sequence of identifiers describing the
-  ///     imported module, which was written in source as a dotted sequence,
-  ///     e.g., `UIKit.UIViewController` will be passed in as the import path
-  ///     array `["UIKit", "UIViewController"]`.
+  ///   - importPath: A nonempty sequence of (token, identifier) pairs
+  ///     describing the imported module, which was written in source as a
+  ///     dotted sequence, e.g., `UIKit.UIViewController` will be passed in as
+  ///     the import path array `[(token, "UIKit"), (token, "UIViewController")]`.
   ///   - version: The version restriction on the imported module. For the
   ///     normal `canImport(<import-path>)` syntax, this will always be
   ///     `CanImportVersion.unversioned`.
   /// - Returns: Whether the module can be imported.
-  func canImport(importPath: [String], version: CanImportVersion) throws -> Bool
+  func canImport(importPath: [(TokenSyntax, String)], version: CanImportVersion) throws -> Bool
```

Comment:
Why do we need both the token and the string? Couldn’t we extract the identifier from the token?

Reply:
The `identifier` on the token is optional, so we'd be asking clients to either force the optional every time, or add a spurious "guard". I'd rather just duplicate the information.

Reply:
Ah, I see. Makes sense. I think that explanation in a doc comment would be useful.